pom.xml

```diff
@@ -5,13 +5,18 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>1.565.3</version>
+		<version>2.2</version>
 	</parent>
 
 	<artifactId>parameterized-scheduler</artifactId>
 	<version>0.3-SNAPSHOT</version>
 	<packaging>hpi</packaging>
 
+	<properties>
+		<jenkins.version>1.642.3</jenkins.version>
```

Comment:
It's quite a big jump. I would be ready to bump to the first pipeline compatible version 1.580.1 without much debate, but for such a bump I would prefer some justification.